website/docs/guides/0.8.2-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,59 @@
+## Issue [INTMDB-175](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/398)
+### resources created with the affected versions ***v0.8.0/v0.8.1***
+
+The following resources were affected, due to a schema update,
+to achieve consistency for multiple providers (AWS) ()[]
```

Comment:
this is missing something, did you mean: 

The following resources were affected, due to a schema update,
to achieve support for multiple providers (AWS and Azure):